app/(site)/temp/all-games/AllGamesDisplay.tsx

```diff
@@ -20,65 +20,55 @@ export default function AllGamesDisplay({
     uid,
     wishlist,
 }: AllGamesDisplayProps) {
-
     return (
-        <div className="space-y-4">
+        <div className="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-6 p-6">
             {games.length > 0 ? (
-                games.map((game) => {
```

Comment:
Same thing here: "I would not remove this, we might need it for later. We should only remove at the end."

Reply:
added back